persona.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+// This protobuf is for adding public/private key pairs to the messaging.
+message persona {
```

Comment:
Capitalize Persona to follow the standard of other protobuf messages

Reply:
Capitalized Persona.